src/components/rules-creation-page/steps/SchedulerStep.jsx

```diff
@@ -31,6 +21,14 @@ const Toggle = ({ label, onChange, value }) => (
 );
 
 const SchedulerStep = ({ syncRule, onChange }) => {
+    const cronExpressions = [
+        { text: i18n.t("Every hour"), value: "0 0 * ? * *" },
```

Comment:
Cool. I don't really understand why the dhis2 team had `apiTranslations.js`, then you end up with two parallel i18n infrastructures...